charts/kcp/values.yaml

```diff
@@ -14,6 +14,12 @@ etcd:
   profiling:
     enabled: false
 kcp:
+  enabled: true
+  sharding:
+    enabled: false
+    isShard: false # set this to true for the all shards, false for the first (root) shard.
+    isRoot: true # set this to true for the first (root) shard, false for all other shards.
```

Comment:
I thought I added a comment here, but apparently I forgot to post it?

I don't think these should be two variables. They are inverse to each other, so `isRoot` should be sufficient (since according to the description, `isShard` should always be `!isRoot`).